eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -59,6 +59,21 @@ steps:
             ${{ parameters.SubscriptionConfiguration }}
           '@ | ConvertFrom-Json -AsHashtable;
 
+          $context = (Get-AzContext)
+          $subscriptionConfiguration["SubscriptionId"] = $context.Subscription.Id
```

Comment:
All the subscription data is in the connection we used to login so we don't need to check-in a configuration file and keep it updated, instead we can just read it from the context.

Reply:
Ah, beautiful